Lmod/branches/7.0/z01-default_modules.csh

```diff
@@ -1,15 +1,17 @@
-if ( ! $?__Init_Default_Modules )  then
-  if ( ! $?LMOD_SYSTEM_DEFAULT_MODULES ) then
-    if ( -f "/usr/share/modulefiles/DefaultModules.lua" ) then
-      setenv LMOD_SYSTEM_DEFAULT_MODULES "DefaultModules"
-    else
-      if ( -f "/usr/share/modulefiles/StdEnv.lua" ) then
-        setenv LMOD_SYSTEM_DEFAULT_MODULES "StdEnv"
+if ( $?USER_IS_ROOT ) then
```

Comment:
my csh-foo is weak, is this really the correct clause? 
I'll assume yes and move on for now